proxy/src/control/discovery.rs

```diff
@@ -56,8 +56,7 @@ pub struct DiscoveryWork<T: HttpService<ResponseBody = RecvBody>> {
 
 struct DestinationSet<T: HttpService<ResponseBody = RecvBody>> {
     addrs: Exists<HashSet<SocketAddr>>,
-    needs_reconnect: bool,
-    rx: UpdateRx<T>,
+    rx: DestinationServiceQuery<T>,
```

Comment:
tioli: might be clearer as `query` or `state` or something, as i usually expect things named `rx` to implement `Stream`

Reply:
> tioli: might be clearer as query or state or something, as i usually expect things named rx to implement Stream

Thanks. I renamed it to `query` like you suggested.